_data/changelogs/modal.yml

```diff
@@ -19,4 +19,45 @@ changelogURL:
   #   githubPr:
   #   githubRepo:
   #   versionUswds:
-# end changelog item
\ No newline at end of file
+
+items:
+- date: 2022-08-05
+  summary: Button - Style aria-disabled to match disabled
+  summaryAdditional: Now disabled styling is applied whether you use disabled (disabled and hidden from screen readers) or aria-disabled (disabled and visible to screen readers). 
+  isBreaking: false
+  affectsAccessibility: true
+  affectsAssets: false
+  affectsGuidance: false
+  affectsJavascript: false
+  affectsMarkup: false
+  affectsStyles: true
+  githubPr: 4783
+  githubRepo: uswds
+  versionUswds: 3.1.0
```

Comment:
Found another item
```suggestion
  versionUswds: 3.1.0
- date: 2022-08-05
  summary: Added <code>type="button"</code> to all non-form buttons to prevent default submit behaviors.
  summaryAdditional: This allowed us to remove <code>preventDefault()</code> from the relevant component JavaScript.
  affectsJavascript: true
  affectsMarkup: true
  githubPr: 4695
  githubRepo: uswds
  versionUswds: 3.1.0
```